### PR TITLE
a11y tests

### PR DESCRIPTION
**Pull request by dtaylor7**
Add a11y tests to build process

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "node-dev": "^3.1.3",
     "nsp": "3.2.1",
     "nyc": "12.0.1",
-    "pa11y": "^5.0.3",
+    "pa11y": "^4.0.1",
```
Comment (mattyod) [inline package.json]: Are there issues with the 5.*.* versions?

File: test/a11y/a11y.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const pa11y = require('pa11y');
+
+module.exports = (testPage, method = 'GET') => {
+  const [host] = /(^http.+:\d+\/)/.exec(testPage);
```
Comment (mattyod) [inline test/a11y/a11y.js]: This runs outside the browser? If so we could perhaps use Nodes' native `new URL.URL(testPage).origin` to make the intent a bit clearer?
https://nodejs.org/api/url.html#url_url_origin

File: test/a11y/index.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+const a11y = require('./a11y');
+const resolveTemplate = require('@hmcts/one-per-page/src/middleware/resolveTemplate');
+
+// ensure step has a template - if it doesnt no need to test it
+const filterSteps = step => {
+  const stepInstance = new step({ journey: {} });
+  return stepInstance.middleware.includes(resolveTemplate);
+};
+
+// ensure step has a template - if it doesnt no need to test it
```
Comment (mattyod) [inline test/a11y/index.js]: Comment doesn't match functions behaviour.

File: package.json (hunk 1 above)
Comment (dtaylor7) [reply package.json]: yea struggled to get it to work so switched to 4 as per petitioner fe

File: test/a11y/index.js (hunk 3 above)
Comment (dtaylor7) [reply test/a11y/index.js]: 👍

File: test/a11y/a11y.js (hunk 2 above)
Comment (dtaylor7) [reply test/a11y/a11y.js]: 👍

File: package.json (hunk 1 above)
Comment (mattyod) [reply package.json]: Maybe we raise a ticket to investigate later.

Comment (dtaylor7) [reply package.json]: sure go for it

Comment (dnenkov) [reply package.json]: I have created a ticket - https://tools.hmcts.net/jira/browse/DIV-2689

**Pull request merged**